Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -270,7 +270,7 @@ def check_input(self, input):
         
 class TGV(RegulariserFunction):
 
-    def __init__(self, alpha=1, gamma=1, max_iteration=100, tolerance=0, device='cpu' , **kwargs):
+    def __init__(self, alpha = 1.0, alpha1 = 1.0, alpha0 = 2.0, max_iteration=100, tolerance=0, device='cpu' , **kwargs):
```

Comment:
This is a backward incompatible change of signature. Why is it needed?

To be absolutely clear, I do not know the reason why `gamma` was chosen to be used and why it is bound between 1 and 2. But now, the user can use `alpha0` and `alpha1` freely, so they may go outside the bound region of their ratio [1,2].

Reply:
I think I changed the signature to match the `FGP_TGV` implementation which solves 

$\min_{u,w} \frac{\lambda}{2}\|\|u-g\|\|^{2} + \alpha_{1}\|\|\nabla u - w\|\| + \alpha_{0}\|\|Ew\|\|$

The ratio is a heuristic choice and it does not always work.

Reply:
It was added here, https://github.com/TomographicImaging/CIL/pull/971#issuecomment-945660804

I'm fine changing the signature, it just needs a justified reason, and it shouldn't break backward compatibility.